src/restconfig/src/main/java/applicationContext.xml

```diff
@@ -512,11 +512,17 @@
             <key><value>/workspaces/{workspace}/coveragestores/{coveragestore}/coverages/{coverage}/index/granules/{granule}</value></key>
             <value>structuredCoverageFinder</value>
         </entry>
+        <entry>
+          <key><value>/sld/{workspace}/{style}</value></key>
+          <value>stylePackageFinder</value>
+        </entry>
```

Comment:
I assume this is the new endpoint, can we do anything more generic (as sld is not the only thing in the mix - see StyleHandler interface).